ons-spark/pyspark-intro/pyspark-intro.ipynb

```diff
@@ -1491,43 +1701,36 @@
     "- [`.write.csv()`](https://spark.apache.org/docs/latest/api/python/reference/api/pyspark.sql.DataFrameWriter.csv.html)\n",
     "- [`.coalesce()`](https://spark.apache.org/docs/latest/api/python/reference/api/pyspark.sql.DataFrame.coalesce.html)\n",
     "- [`.mode()`](https://spark.apache.org/docs/latest/api/python/reference/api/pyspark.sql.DataFrameWriter.mode.html)\n",
     "\n",
-    "Spark in ONS material:\n",
-    "- Introduction to PySpark: Exercises\n",
-    "- Understanding and Optimising Spark chapter\n",
-    "- Guidance on Spark sessions\n",
-    "- Sample Spark Sessions\n",
-    "- Introduction to PySpark\n",
-    "- F.col()\n",
-    "- Parquet files\n",
-    "- Optimising Joins\n",
-    "- Output File Choices\n",
-    "- Writing Data\n",
-    "- Pydoop\n",
-    "\n",
-    "Other links:\n",
-    "- [Python documentation](https://docs.python.org)\n",
-    "    - [Python Tutorial](https://docs.python.org/3/tutorial/)\n",
-    "    - [`dir()`](https://docs.python.org/3/library/functions.html#dir)\n",
-    "    - [`help()`](https://docs.python.org/3/library/functions.html#help)\n",
-    "    - [`os.remove()`](https://docs.python.org/3/library/os.html#os.remove)\n",
-    "    - [`shutil.rmtree()`](https://docs.python.org/3/library/shutil.html#shutil.rmtree)\n",
-    "    - [`subprocess.run()`](https://docs.python.org/3/library/subprocess.html#subprocess.run)\n",
-    "- [`pandas` documentation](https://pandas.pydata.org/):\n",
-    "    - [10 Minutes to pandas](https://pandas.pydata.org/pandas-docs/stable/user_guide/10min.html)\n",
-    "    - [`.to_csv()`](https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.to_csv.html) \n",
+    "Python Documentation\n",
+    "- [Python Tutorial](https://docs.python.org/3/tutorial/)\n",
+    "- [`dir()`](https://docs.python.org/3/library/functions.html#dir)\n",
+    "- [`help()`](https://docs.python.org/3/library/functions.html#help)\n",
+    "- [`os.remove()`](https://docs.python.org/3/library/os.html#os.remove)\n",
+    "- [`shutil.rmtree()`](https://docs.python.org/3/library/shutil.html#shutil.rmtree)\n",
+    "- [`subprocess.run()`](https://docs.python.org/3/library/subprocess.html#subprocess.run)\n",
+    "\n",
+    "pandas Documentation:\n",
+    "- [10 Minutes to pandas](https://pandas.pydata.org/pandas-docs/stable/user_guide/10min.html)\n",
+    "- [`.to_csv()`](https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.to_csv.html) \n",
+    "    \n",
+    "Other Links:\n",
+    "- [Databricks: Learning Spark](https://pages.databricks.com/rs/094-YMS-629/images/LearningSpark2.0.pdf)\n",
     "- [Wikipedia; Join (SQL)](https://en.wikipedia.org/wiki/Join_(SQL))\n",
-    "- [Databricks: Learning Spark](https://pages.databricks.com/rs/094-YMS-629/images/LearningSpark2.0.pdf)"
+    "\n",
+    "#### Acknowledgments\n",
+    "\n",
+    "Thanks to Karina Marks, Chris Musselle and Dave Beech for creating the initial version of this article."
```

Comment:
Did Dave Beech contribute towards this?

Reply:
When I looked at the previous commits he had a small one to fix some typos.